flink-doris-connector/src/main/java/org/apache/doris/flink/sink/schema/SchemaChangeManager.java

```diff
@@ -68,6 +71,12 @@ public SchemaChangeManager(DorisOptions dorisOptions, String charsetEncoding) {
     }
 
     public boolean createTable(TableSchema table) throws IOException, IllegalArgumentException {
+        // auto create database if not exists
+        if (!checkDatabaseExists(table.getDatabase())) {
+            execute(
+                    SchemaChangeHelper.buildCreateDatabaseDDL(table.getDatabase()),
+                    SchemaChangeHelper.DEFAULT_DATABASE);
+        }
```

Comment:
This branch seems to have no case coverage?

Reply:
done